website/src/plugins/github-service.spec.ts

```diff
@@ -0,0 +1,141 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { http, HttpResponse } from 'msw';
+import { setupServer } from 'msw/node';
+import { afterAll, afterEach, beforeAll, beforeEach, describe, expect, test } from 'vitest';
+
+import type { GitHubMetadata } from './github-metadata';
+import { GitHubService } from './github-service';
+import { mockReleaseData } from './test-resourcecs/mock-release-data';
+
+const URL = 'https://api.github.com/repos/podman-desktop/podman-desktop/releases/latest';
+
+const server = setupServer(
+  http.get(URL, () => {
+    return HttpResponse.json(mockReleaseData);
+  }),
+);
+
+describe('GitHubService', () => {
+  const owner = 'podman-desktop';
+  const repo = 'podman-desktop';
```

Comment:
I think it should be constants because 'podman-desktop' for org and 'podman-desktop' for repository name are like duplicated tons of times across these new files

let say tomorrow I change the org name, I should not have to change at multiple places